### DISCUSSION TITLE
Next.js public roadmap

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/next.js#11443] [Help]
Is there a public roadmap somewhere?

If so, is it possible to add some ROADMAP.md in the repo that point towards whatever exists?

Having an overview of where the framework is going would be nice, a 3-6 months visibility would be great.

**Comment by romainquellec** [comment]
I suggested that 2 years ago, and I still can't find it today. I guess we will never see it.
Still, we can check that from time to time : https://github.com/vercel/next.js/milestones
